source/install/cloud-changelog.md

```diff
@@ -4,12 +4,39 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-06-22](#release-2022-06-22)
 - [Release 2022-06-15](#release-2022-06-15)
 - [Release 2022-06-14](#release-2022-06-14)
 - [Release 2022-05-26](#release-2022-05-26)
 - [Release 2022-05-12](#release-2022-05-12)
 - [Release 2022-04-28](#release-2022-04-28)
-- [Release 2022-04-13](#release-2022-04-13)
+
+## Release 2022-06-22
+
+### Improvements
+
+#### User Interface (UI)
+ - Recent emojis are now sorted based on the number of times an emoji has been used.
+ - Improved the link preview user interface.
+ - Added new search shortcuts to the **Keyboard Shortcuts** modal. 
+    - CMD+F (macOS)
+    - CTRL+F (Windows) Web App (Browser)
+    - CMD+SHIFT+F (macOS)
+    - CTRL+SHIFT+F (Windows)
```

Comment:
What's the difference in functionality between the keyboard shortcuts that use SHIFT versus the ones that don't?